appveyor.yml

```diff
@@ -4,10 +4,9 @@ skip_tags: true
 
 environment:
   matrix:
-    - nodejs_version: "6"
     - nodejs_version: "8"
```

Comment:
Is Node 8 still being supported?
Looks like Travis is testing 10 and 12
https://github.com/remarkjs/remark-git-contributors/blob/c23d36bd911aa0c40edb43378d1653db35247a2f/.travis.yml#L3-L4

Reply:
Yes, node 8 goes EOL in december

Reply:
What's the support policy of remarkjs, if any?

Reply:
There is no official node version support policy that I'm aware of.
Currently Unified itself supports Node 10 and Node 12

Reply:
OK. I'd like to request (for this module at least, but if possible also for the rest of the ecosystem) that we keep node 8 support. Dropping versions too early creates a wave of dependents having to upgrade.

Reply:
> I'd like to request for this module [...], we keep node 8 support

Ok, I'd be fine with keeping Node 8 in the AppVeyor matrix for now.

> also for the rest of the ecosystem

Dropping Node 8 testing was one of the breaking changes for the Unified 8 major release, and is trickling out the rest of the ecosystem. (:notebook: Node 8 may still work for some time, but is not officially supported)
I don't think walking that back would be a good idea at this point.

That being said, what you may be getting at is, the want/need for a better defined Node.js support policy.
Similar to the request in https://github.com/unifiedjs/unified/issues/70
That could be something worth raising in https://github.com/unifiedjs/ideas

> Dropping versions too early creates a wave of dependents having to upgrade.

Dropping version support at any time causes a wave of dependents to have to upgrade, early or not.
Again, what you may be getting at is, the want/need for a better defined Node.js support policy.
This could be something worth raising in https://github.com/unifiedjs/ideas

Reply:
:notebook: this discussion is also tangentially related to https://github.com/unifiedjs/unified/pull/73#issuecomment-537608479

Reply:
The later you drop a node version, the more likely it is that dependents have already upgraded. That reduces friction. Many projects follow the Node.js schedule, which is a simple policy. The ideal policy though - from a consumers perspective - is not dropping a version until you really have to. In `remark-git-contributors` there's no code that actually needs node >= 10.

> what you may be getting at is, the want/need for a better defined Node.js support policy.

Would help, yes. This module used to have an `engines` field in `package.json` which I think is a more obvious indication of support than the versions listed in `travis.yml`.

Reply:
@vweevers The PR seems to drop older versions, yet your comments seem to go against that? 🤔

Reply:
The above discussion is not really relevant to this PR. It drops node 6, not 8. I opened this PR because I saw that one of our dependencies requires node 8, and the tests can't pass without it.

Reply:
OK, think the discussion then makes more sense in the other thread on unified!